influxdb/files/influxdb.conf

```diff
@@ -243,7 +243,7 @@ reporting-disabled = {{ server.reporting_disabled | lower }}
 ### Controls one or many listeners for collectd data.
 ###
 
-[[collectd]]
+[collectd]
```

Comment:
The current syntax is valid:
https://docs.influxdata.com/influxdb/v1.3/administration/config/#collectd

You're probably using an old version of InfluxDB.